controller/handler/util_test.go

```diff
@@ -2,6 +2,8 @@ package handler
 
 import (
 	"fmt"
+	"github.com/TwiN/gatus/v5/config"
+	"github.com/TwiN/gatus/v5/storage"
```

Comment:
The imports should be split in two groups: imports from the standard library, and external imports.

In other words, this should be 
```go
import (
	"fmt"
	"net/http"
	"testing"

	"github.com/TwiN/gatus/v5/config"
	"github.com/TwiN/gatus/v5/storage"
)
```